AHBC_MIDTERM_2019_JULY_TEAMROCKET/Program.cs

```diff
@@ -1,12 +1,59 @@
 ﻿using System;
+using System.IO;
 
 namespace AHBC_MIDTERM_2019_JULY_TEAMROCKET
 {
     class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            var paymentType = new PaymentType();
```

Comment:
It looks like you were testing this in the main class, but maybe we remove this for the upstream file and leave the program file empty for now?